### PR TITLE
Bugfix -  Relative file path calculation in plan runner

### PR DESCRIPTION
**Pull request by tronxd**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checkov/terraform/plan_runner.py [hunk 1]
```diff
@@ -59,19 +59,19 @@ def run(self, root_folder=None, external_checks_dir=None, files=None, runner_fil
                         continue
                     self.tf_definitions = tf_definitions
                     self.template_lines = template_lines
-                    self.check_tf_definition(report, runner_filter)
+                    self.check_tf_definition(report,root_folder, runner_filter)
```
Comment (nimrodkor) [inline checkov/terraform/plan_runner.py:62]: @tronxd Please format this file

File: Pipfile [hunk 2]
```diff
@@ -28,7 +28,7 @@ dpath = ">=1.5.0,<2"
 pyyaml = ">=5.4.1"
 boto3 = "==1.17.*"
 GitPython = "*"
-six = "==1.15.0"
+six = "==1.16.0"
```
Comment (nimrodkor) [inline Pipfile:31]: @tronxd Why is this required?

**Review by nimrodkor — COMMENTED** [review]
@tronxd Seems to be breaking the `test_plan_runner` tests. Can you fix those?

File: Pipfile (hunk 2 above)
Comment (tronxd) [reply Pipfile:31]: @nimrodkor It has a conflict in pipenv when trying to install with 1.15 (guess that six 1.16 is pinned in one of our deps)

**Review by nimrodkor — CHANGES_REQUESTED** [review]
I prefer not to modify the behavior of the existing runners. Can we avoid that?
